test/lambdaconnect_model/scoping_test.cljc

```diff
@@ -104,84 +113,16 @@
-                                                               [?LATicketsSold-byUserEvent :LATicketsSold/game ?LAGame-organisedByUser]
-                                                               (or-join [?user ?LAGame-organisedByUser]
-                                                                        (and [?LAGame-organisedByUser :LAGame/organiser ?LAUser-me]
-                                                                             [?LAUser-me :LAUser/internalUserId ?G__28253]
-                                                                             [?user :app/uuid ?G__28253])
-                                                                        (and [?LAGame-organisedByUser :LAGame/inThePast true]
-                                                                             [?user :user/username]))
-                                                               [?LAUser-me :LAUser/internalUserId ?G__28253]
-                                                               [?user :app/uuid ?G__28253]]
-                                  :LATeam.playedInGame '[:find
-                                                         ?user
-                                                         :in
-                                                         $
-                                                         [?LATeam-playedInGame ...]
-                                                         :where
-                                                         [?LATeam-playedInGame :LATeam/playedIn ?LAGame-organisedByUser]
-                                                         (or-join [?user ?LAGame-organisedByUser]
-                                                                  (and [?LAGame-organisedByUser :LAGame/organiser ?LAUser-me]
-                                                                       [?LAUser-me :LAUser/internalUserId ?G__8960]
-                                                                       [?user :app/uuid ?G__8960])
-                                                                  (and [?LAGame-organisedByUser :LAGame/inThePast true]
-                                                                       [?user :user/username]))]
-                                  :LASyncInfo.system '[:find ?user :in $ [?LASyncInfo-system ...] :where [?user :user/username]]
-                                  :LACnysInfo.system '[:find ?user
-                                                       :in $ [?LACnysInfo-system ...]
-                                                       :where
-                                                       [(identity ?LACnysInfo-system) ?user]
-                                                       [(!= ?LACnysInfo-system ?user)]]}]
+                                               [(!= ?user ?LACnysInfo-system)]
+                                               [(identity ?user) ?LACnysInfo-system]]}]
```

Comment:
`?LACnysInfo-system` nie jest zdefiniowane w pierwszej klauzuli. Ciekawe, że Datomic rozumie takie zapytanie (sprawdziłem i działa, najwyraźniej Datomic robi wewnętrzny reordering, który to naprawia).